lib/bali/form_builder/radio_fields.rb

```diff
@@ -16,8 +16,65 @@ def radio_field(method, values, options = {}, html_options = {})
         field_helper(method, field, options)
       end
 
+      def radio_buttons_grouped(
```

Comment:
En todos los form builders tenemos 1 método con el input solamente y luego agregamos una version con label y mensajes de error. (en este caso podria ser `radio_buttons_field_group`)

Que te parece? Para mantener consistencia con los demas tipos de form builder inputs

```suggestion
      def radio_buttons_field(
```